src/BenchmarksApps/TechEmpower/RazorPages/RazorPages.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <LangVersion>preview</LangVersion>
+    <UserSecretsId>38063504-d08c-495a-89c9-daaad2f60f31</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.123" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.0-preview.4" />
```

Comment:
I believe the other projects have conditional references based on the 7/8 TFM, at least mvc.csproj does.

Reply:
True, I copied BlazorSSR but that's net8.0 only so this one could be 7.0 too.

Reply:
Ok, it's just that the default tfm in the csproj is 7.0 currently